build-tools/scripts/jdk.mk

```diff
@@ -33,16 +35,17 @@ endif # 32-bit
 
 # Darwin supports three possible search locations:
 #
-# 1. `/Library/Java/JavaVirtualMachines/jdk*`
+# 1. `/Library/Java/JavaVirtualMachines/jdk1.*`
```

Comment:
Is this filtering correct? On Linux I have I have `/usr/lib/jvm/java-8-openjdk-amd64` which would be filtered out by this if this were also applied to Linux.

Reply:
This is within an `ifeq ($(OS),Darwin)` block, so this change won't impact Linux.

Reply:
I keep writing "if it were... on Linux".

Reply:
I think @atsushieno is asking if `1.8` will work on Linux since his folder says `8`.

Looked on ubuntu and I have symbolic links:
```
$ ls -la /usr/lib/jvm/
...
lrwxrwxrwx   1 root root   20 Oct 23 17:43 java-1.8.0-openjdk-amd64 -> java-8-openjdk-amd64
lrwxrwxrwx   1 root root   20 Aug 24 12:15 java-1.9.0-openjdk-amd64 -> java-9-openjdk-amd64
drwxr-xr-x   5 root root 4096 Dec 18 22:58 java-8-openjdk-amd64
drwxr-xr-x   8 root root 4096 Dec 18 23:00 java-9-openjdk-amd64
```

I just did `apt install` `open-jdk-8-headless` and `open-jdk-9-headless`.

But still, I need to:
- Make the Java 9 restriction an option that can be passed in by `xamarin-android`
- Fix this for Linux, too
- On Windows, `JdkInfo.props` is generated by an MSBuild task in `xamarin-android`, so I won't have to deal with it until then